src/policy/fees.cpp

```diff
@@ -660,10 +657,8 @@ CFeeRate CBlockPolicyEstimator::estimateRawFee(int confTarget, double successThr
         stats = longStats.get();
         break;
     }
-    default: {
-        throw std::out_of_range("CBlockPolicyEstimator::estimateRawFee unknown FeeEstimateHorizon");
-    }
-    }
+    } // no default case, so the compiler can warn about missing cases
+    assert(stats);
```

Comment:
nit, `assert(stats)` seems to be checking for more than missing cases (like `assert(false)`), maybe update the comment

Reply:
I think the comment and `assert(stats);` are independent here, as `std::unique_ptr::get` could return `nullptr`.

Reply:
You're right, nvm